Submitted By: Axel Bergerhoff (axelbergerhoff .a.t. compuserve .d.o.t. com)
Date: 2009-07-02
Initial Package Version: 22.8
Origin: Axel Bergerhoff
Upstream Status: Not submitted - LFS Specific
Description: Allow building with gcc-4.x.y


diff -Naur lilo-22.8.orig/checkit lilo-22.8/checkit
--- lilo-22.8.orig/checkit	2006-08-13 21:24:45.000000000 +0200
+++ lilo-22.8/checkit	2009-07-02 11:41:13.000000000 +0200
@@ -44,8 +44,8 @@
 
 #echo
 echo GCC version 2.91 or later is required
-gcc -v 1>foo1 2>foo2
-V=`cat foo1 foo2 | grep version | cut -d " " -f 3`
+gcc --version 1>foo1 2>foo2
+V=`cat foo1 foo2 | grep gcc | cut -d " " -f 3`
 rm -f foo1 foo2
 if [ -z $V ]; then
 	echo gcc is not present
